### PR TITLE
Bugfix/maven classpath schema

### PR DESCRIPTION
**Pull request by vtitov**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh`, `./bin/security/{LANG}-petstore.sh` and `./bin/openapi3/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

https://github.com/OpenAPITools/openapi-generator/issues/2408

### CODE REVIEW

**Comment by vtitov** [comment]
AppVeyor build failed with suspicious message. I believe it's infrastructure issue:
> Build started
> git clone -q https://github.com/OpenAPITools/openapi-generator.git C:\projects\openapi-generator
> fatal: unable to access 'https://github.com/OpenAPITools/openapi-generator.git/': Could not resolve host: github.com
> Command exited with code 128

**Comment by wing328** [comment]
@vtitov thanks for the PR. Can you name "swagger.yaml" as "openapi.yaml" instead?

For the AppVeyor failure, don't worry about it.

**Comment by wing328** [comment]
Your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

File: modules/openapi-generator-maven-plugin/examples/multi-module/java-client/pom.xml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                </dependencies>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <!-- specify the swagger yaml -->
+                            <!--<inputSpec>${project.basedir}/swagger.yaml</inputSpec>-->
+                            <inputSpec>swagger.yaml</inputSpec>
```
Comment (wing328) [inline modules/openapi-generator-maven-plugin/examples/multi-module/java-client/pom.xml]: @vtitov Can we also replace swagger.yaml with openapi.yaml in line 36 and 37?

Comment (vtitov) [reply modules/openapi-generator-maven-plugin/examples/multi-module/java-client/pom.xml]: @wing328, thank you!
I changed java-client/pom.xml and made actions to links commits to my email.

**Pull request merged**